### ISSUE TITLE
Providing a Slack Channel ID for trigger target returns an error.

### ISSUE THREAD

**Issue by scgerkin** [honeycombio/terraform-provider-honeycombio#533]
We're using Terraform to manage a few Honeycomb triggers and recently changed a channel name which necessitated needing to update the trigger targets. After testing in the UI, I was able to successfully send a test trigger to the new channel by using the Channel ID rather than the `#channel-name`. However, the Terraform provider doesn't like this:

```tf
Error: Error Updating Honeycomb Trigger

  on k8s_triggers.tf line 206, in resource "honeycombio_trigger" "k8s_pods_pending_spike":
 206: resource "honeycombio_trigger" "k8s_pods_pending_spike" {

recipients: (0: (target: slack target must begin with '#' or '@'.).).
```

**Versions**

0.18.0 and 0.26.0

**Steps to reproduce**

Attempt creating a `honeycombio_trigger` as:
```tf
resource "honeycombio_trigger" "my_trigger" {
  name        = "My trigger"
  description = "My desc"

  query_id = honeycombio_query.my_query.id
  dataset  = "my-dataset"

  frequency = 60

  alert_type = "on_change"

  threshold {
    op    = ">="
    value = some_val
  }

  recipient {
    type   = "slack"
    target = "C1234"
  }
}
```

**Additional context**

n/a

**Comment by jharley** [comment]
Hey, @scgerkin! Thanks for raising this.

We're aware of this issue (and discrepancy with the UI) and are currently working on improving the Slack integration. This limitation is in the underlying API -- not just the provider -- is the scope of our work.

**Comment by jharley** [comment]
Support for providing channel IDs was just released in v0.27.1
